src/Illuminate/Foundation/Console/ServeCommand.php

```diff
@@ -77,9 +77,23 @@ protected function serverCommand()
      */
     protected function host()
     {
+        if ($this->input->getOption('lan')) {
+            return $this->lanHost();
+        }
+
         return $this->input->getOption('host');
     }
 
+    /**
+     * Get the LAN host for the command.
+     *
+     * @return string
+     */
+    protected function lanHost(): string
+    {
+        return explode(PHP_EOL, shell_exec('ifconfig | grep "inet " | grep -Fv 127.0.0.1 | awk \'{print $2}\''))[0];
```

Comment:
That might easily break.

Reply:
Also this will just flat-out not work on Windows.